src/tests/Microsoft.PowerFx.Core.Tests.Shared/DisplayNameTests.cs

```diff
@@ -787,5 +788,39 @@ public void RenameParameterTextFirst(string expr, string expected, bool find)
                 Assert.Equal(expected, renamer.ApplyRename(expr));
             }
         }
+
+        [Fact]
+        public void SummarizeDisplayNameTest()
```

Comment:
@jas-valgotar This is your original test from https://github.com/microsoft/Power-Fx/pull/2801.